demo/bootsfaces-applicant-portlet/src/main/webapp/WEB-INF/views/applicant.xhtml

```diff
@@ -105,7 +106,7 @@
 						<h:outputText value="#{i18n['attachments']}" />
 					</f:facet>
 					<h:form id="f2" enctype="multipart/form-data" prependId="false">
-						<b:modal id="modal">
+						<b:modal id="modal" style="display: none;">
```

Comment:
@stiemannkj1 Would this change be more appropriately fixed under https://issues.liferay.com/browse/FACES-3151? Also, it is only a Liferay Portal problem right? If it applies to all modals, then perhaps it would be more appropriate to specify in a CSS file?